owl/EASE-OBJ.owl

```diff
@@ -1876,6 +1885,16 @@
     
 
 
+    <!-- http://www.ease-crc.org/ont/EASE-OBJ.owl#FunctionalDesign -->
+
+    <owl:Class rdf:about="http://www.ease-crc.org/ont/EASE-OBJ.owl#FunctionalDesign">
+        <rdfs:subClassOf rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Design"/>
+        <rdfs:comment>The design of an object from functionality point of view. A functional design is useful to develop complex modular objects with components that have a specific purpose, and can function with minimum side effect on other components of that object. </rdfs:comment>
```

Comment:
I do not get the point about "minimum side effect on other components". You mean that a functional design restricts how functional components interplay with each other, and thereby ensures that they can function with minimum side effects?
Is there an example of a side effect you have in mind?

Reply:
it can describe the functional responsibilities of individual components such a way that in complex systems/devices, functioning of individual components would not lead interference among each other.
https://en.wikipedia.org/wiki/Functional_design#:~:text=Functional%20Design%20is%20a%20paradigm,side%20effects%20on%20other%20parts.

Reply:
not a good reference :P

Reply:
so the text written there is also unclear in my opinion. It is also marked as bad page because there are missing references on that wikipedia page

Reply:
we can fine tune this comment later on